packages/caip-stream/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './createCaipStream';
```

Comment:
Nit: One convention we've been following is to be explicit about named exports in `index` files. I know that there is only one here but if we ever add more in the future it would be good to know exactly what is being exported, and this file tells us just that. 

What about:

``` suggestion
export { createCaipStream } from './createCaipStream';